packages/popover/src/popover.ts

```diff
@@ -44,6 +55,39 @@ export class Popover extends LitElement {
         `;
     }
 
+    public connectedCallback(): void {
+        super.connectedCallback();
+        this.addEventListener('sp-overlay-query', this.onOverlyQuery);
+    }
+
+    public disconnectedCallback(): void {
+        super.disconnectedCallback();
+        this.removeEventListener('sp-overlay-query', this.onOverlyQuery);
+    }
+
+    public onOverlyQuery(event: CustomEvent<OverlayDisplayQueryDetail>): void {
+        if (!event.target || !this.shadowRoot) return;
+
+        const target = event.target as Node;
+        if (!target.isSameNode(this)) return;
+
+        const tipElement = this.shadowRoot.querySelector('#tip') as HTMLElement;
+        if (tipElement) {
+            event.detail.overlayContentTipElement = tipElement;
+        }
+    }
+
+    updated(changedProperties: PropertyValues): void {
+        if (
+            changedProperties.has('placement') ||
+            changedProperties.has('dataPopperPlacement')
+        ) {
+            this.computedPlacement =
+                spectrumPlacement(this.dataPopperPlacement || this.placement) ||
+                'top';
+        }
```

Comment:
If we modify the overlay implementation to rename the `data-popper-placement` attribute on content, then maybe we can make this attribute name configurable, and do our spectrumPlacement translation there?

Reply:
I removed the need for `data-popper-placement` and I added a custom transform to the CSS when processing to loosen up the matching of the `placement` attribute by changing the operator from `=` to `*=`